templates/chat/roll/test-card.html

```diff
@@ -54,7 +54,29 @@
             - <span class ="critical-roll"> {{test.result.criticalchannell}}</span>
             {{/if}}
             {{#if test.result.fumble}}
-            - <a class ="table-click fumble-roll" data-table = "oops"><i class="fas fa-list"></i> {{test.result.fumble}}</a>
+            <ul>
+                {{#if (tableExists "oops")}}
+                <li><a class ="table-click fumble-roll" data-table = "oops"><i class="fas fa-list"></i> {{test.result.fumble}} (Podręcznik)</a></li>
+                {{/if}}
+                {{#if (tableExists "oopsarrows")}}
+                <li><a class ="table-click fumble-roll" data-table = "oopsarrows"><i class="fas fa-list"></i>{{test.result.fumble}} (Strzelecka)</a></li>
+                {{/if}}
+                {{#if (tableExists "oopsblackpowder")}}
+                <li><a class ="table-click fumble-roll" data-table = "oopsblackpowder"><i class="fas fa-list"></i>{{test.result.fumble}} (Prochowa)</a></li>
+                {{/if}}
+                {{#if (tableExists "oopsthrowing")}}
+                <li><a class ="table-click fumble-roll" data-table = "oopsthrowing"><i class="fas fa-list"></i>{{test.result.fumble}} (Miotana)</a></li>
+                {{/if}}
+                {{#if (tableExists "oopsblades")}}
+                <li><a class ="table-click fumble-roll" data-table = "oopsblades"><i class="fas fa-list"></i>{{test.result.fumble}} (Sieczna)</a></li>
+                {{/if}}
+                {{#if (tableExists "oopspummelling")}}
+                <li><a class ="table-click fumble-roll" data-table = "oopspummelling"><i class="fas fa-list"></i>{{test.result.fumble}} (Obuchowa)</a></li>
+                {{/if}}
+                {{#if (tableExists "oopsbrawling")}}
+                <li><a class ="table-click fumble-roll" data-table = "oopsbrawling"><i class="fas fa-list"></i>{{test.result.fumble}} (Bijatyka)</a></li>
+                {{/if}}
+            </ul>
```

Comment:
I'm not sure I like this (and there's still some Polish here)

The way I've seen people do this before is use multi-columned oops tables, so when you click on the table links, it lists the columns to roll on, instead of crowding the test card with all these links.

Reply:
ok, i checked the tableroll method and play around with it, for now it was removed, thanks